### PR TITLE
Rename metric name for slow requests

### PR DESCRIPTION
**Pull request by ctrlok**
It was a great surprise for us that last bucket also contain all slow requests :)

### CODE REVIEW

**Comment by nnuss** [comment]
At 722
```golang
// +1 to track every over the number of buckets we track
timeBuckets = make([]int64, Config.Buckets+1)
```
should no longer be +1.

By doing this we lose `&target=carbon.zipper.{host}.requests_in*` to plot them all together trivially - and gain a more honest name than `...requests_in_900ms_to_1000ms` (for default buckets of 10).

File: main.go [hunk 1]
```diff
@@ -717,7 +719,7 @@ func main() {
 	}
 
 	// +1 to track every over the number of buckets we track
```
Comment (nnuss) [inline main.go]: [nit] would you remove the comment that's no longer relevant too ? :)

Comment (ctrlok) [inline main.go]: excuse me :) done

**Comment by nnuss** [comment]
I chatted with @dgryski today and we agreed that the name should change. We'd like to maintain as long of a prefix match as possible to make for nice globbing and minimal dashboard changes. I think keeping the lower edge of the time bucket (900) in the name might also be useful.

```
requests_in_900ms_to_infinity  // not so bad
requests_in_900ms_to_+INFms // math.Inf(1) could be problematic

requests_in_900ms_and_later
requests_in_900ms_and_longer
requests_in_900ms_and_over

// first word after "requests" should be lexically after 'in' sort to the end
// this is also *slightly* wrong as a '900ms' request would end up in this bucket
requests_longer_than_900ms
requests_over_900ms
requests_slower_than_900ms
```

Any of ^^^ those appeal or another suggestion?

**Comment by ctrlok** [comment]
`requests_in_900ms_to_infinity`looks good for me.

**Comment by nnuss** [comment]
(@dgryski agreed also)
Would you update the name?
Then I think it's good to merge.

**Comment by nnuss** [comment]
(missed your edit)
> But it's hard o manage slow requests. I mean name of metric will change if we change number of buckets.

True; if we grow from 10 -> 20 for example:
...
`carbon.zipper.host.requests_in_900ms_to_1000ms`
`carbon.zipper.host.requests_in_900ms_to_infinity` // stops being sent
`carbon.zipper.host.requests_in_1000ms_to_1100ms` // already a lexical sort annoyance :p
...
`carbon.zipper.host.requests_in_1900ms_to_infinity`

the `...requests_in_900ms_to_infinity` stops being updated at the same time the metrics for the new buckets appear.

---

If we instead had something static like:
`carbon.zipper.host.requests_that_were_slow`

Then when we change the number of buckets the meaning of the value of that metric changes while the name doesn't and we don't orphan a series.

---

I view both as having downsides. I guess I find the dead-metric/cleanup in the first scenario (_infinity) preferable to the meaning change in the second (_that_were_slow).

**Pull request closed without merging**